portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase

```diff
@@ -752,4 +752,100 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-133280. Publishing a content page when staging is enabled, the changes must be reflected on the page."
```

Comment:
Should be This is a test for LPS-130482. View the custom CSS of content page in staging site.